eve/scripts/release

```diff
@@ -105,5 +105,16 @@ for chart in CHARTS:
         args.version, [zenko_version['minor'], patch], verbose=args.verbose)
     save_file(chart, chart_path)
     save_file(values, values_path)
+
 # update Zenko
-update_chart(open_file('%s/Chart.yaml' % args.path, verbose=args.verbose), None, args.version, None, verbose=args.verbose, dep=False)
\ No newline at end of file
+chart_path = '%s/Chart.yaml' % (args.path)
+values_path= '%s/values.yaml' % (args.path)
+chart, values = update_chart(open_file(chart_path, verbose=args.verbose),
+                             open_file(values_path, verbose=args.verbose),
+                             args.version,
+                             None,
+                             verbose=args.verbose,
+                             dep=False)
+# only update Charts.yaml
+save_file(chart, chart_path)
+# save_file(values, values_path)
```

Comment:
Why not saving the values file?

Reply:
I had skipped that because `zenko/values.yaml` does not contain the root-level `image.*` fields. (It seems to be specifically written for updating the dependencies in `zenko/charts/*`

Reply:
alrighty

Reply:
thanks for that info

Reply:
Now that I think about it a little bit. Should we remove the commented line then?